frontend/app/views/welcome.scala.html

```diff
@@ -16,7 +16,7 @@ <h3 class="feature-panel__description">
     </div>
 }
 
-@main("Home Page") {
+@main(pageInfo.title, pageInfo=pageInfo) {
```

Comment:
This is a bit of an oddity we have `title` in `PageInfo` but also pass it manually as a string in the template. Should probably try and remove this duplication at some point.